MIGRATING.md

```diff
@@ -74,9 +74,8 @@ Rename environment variables:
 | ---------------------------------- | ------------------------------------ | ----- |
 | SIGNALFX_ACCESS_TOKEN              | SPLUNK_ACCESS_TOKEN                  |       |
 | SIGNALFX_SERVICE_NAME              | OTEL_SERVICE_NAME                    |       |
-| SIGNALFX_ENDPOINT_URL              | OTEL_EXPORTER_JAEGER_ENDPOINT        | if Jaeger is used |
-| SIGNALFX_ENDPOINT_URL              | n/a                                  | OTLP is not implemented yet |
-| SIGNALFX_RECORDED_VALUE_MAX_LENGTH | SPLUNK_MAX_ATTR_LENGTH               |       |
+| SIGNALFX_ENDPOINT_URL              | OTEL_EXPORTER_OTLP_ENDPOINT          |       |
```

Comment:
This is not a 1-1 mapping. We should add comment that this needs to be OTLP endpoint OR they need to use `OTEL_EXPORTER_JAEGER_ENDPOINT`

Reply:
True, but aren't we asusming they're migrating to Splunk APM which, in a way, is an equivalent with OTLP-support?

Reply:
I added https://github.com/signalfx/splunk-otel-js/pull/218/commits/7d3db551434cc4c063455d46494f1e01926b2700 for now, but rather remove it regarding my comment above.